bindsnet/analysis/plotting.py

```diff
@@ -416,7 +417,7 @@ def plot_performance(
 
     for scheme in performances:
         ax.plot(
-            range(len(performances[scheme])),
+            [n * x_scale for n in range(len(performances[scheme]))],
```

Comment:
Instead of having x_scale, can we just pass an array as standard plotting libraries do? It was it is more flexible and allows us to handle different scales like log/ exponential for example?

Reply:
hmm, for the sake of keeping eth simple for now I'm hesitant but I think it's a good point. It seems hard to me to add this axis without adding a new param which I'd like to avoid. If you have a suggestion though it would be good to have an axis like that

Reply:
also we could still add to plotting.py to get log or exp scaling, we would just need to change the plt scale, it still is an array just scaled by a factor to make it a literal count of examples

Reply:
![Figure_2](https://user-images.githubusercontent.com/44758003/75646722-8f37f100-5c18-11ea-99a4-57f76737a3e8.png)

for now at least I'd like to add this change for myself

Reply:
What I am suggesting is something like matplotlib implementation:

> x, y : array-like or scalar
The horizontal / vertical coordinates of the data points. x values are optional and default to range(len(y)).

>Commonly, these parameters are 1D arrays.

>They can also be scalars, or two-dimensional (in that case, the columns represent separate data sets).

>These arguments cannot be passed as keywords.

Reply:
I think I like @dee0512's suggestion: just pass in what you want to plot. But I think I'm happy either way. @dee0512 do you feel strongly one way or another about this change?

Reply:
No really unless we plan to use the function more in the future. The plotting of bindsnet is not used much right now. 

But making it more general might help that.